commit.go

```diff
@@ -244,7 +244,8 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 	}
 	if exporting {
 		// Copy everything.
-		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
+		// TOM: todo add credsContext
+		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter, nil))
```

Comment:
I'd think we'd want to set creds here to cover cases where we're committing directly to a registry.